drivers/gles3/shaders/scene.glsl

```diff
@@ -768,17 +762,18 @@ void light_compute(vec3 N, vec3 L, vec3 V, float A, vec3 light_color, float atte
 	alpha = min(alpha, clamp(1.0 - attenuation, 0.0, 1.0));
 #endif
 
-#endif //defined(LIGHT_CODE_USED)
+#endif // LIGHT_CODE_USED
 }
 
-float get_omni_attenuation(float distance, float inv_range, float decay) {
+float get_omni_spot_attenuation(float distance, float inv_range, float decay) {
```

Comment:
That's the only other code change, just a rename because this is used by both Omni and Spot paths. Happy to revert back if this is still an "omni" attenuation and its use in spot shouldn't be reflected in the name.